### ISSUE TITLE
To String

### ISSUE THREAD

**Issue by sirinath** [fastfloat/fast_float#52]
Perhaps fast string conversion also can be added.

**Comment by lemire** [comment]
Is this related to https://github.com/fastfloat/fast_float/issues/23 ?

**Comment by sirinath** [comment]
Yes. But either one round trip (float/decimal -> string -> float/decimal & string -> float/decimal -> string) implementations will to.

**Comment by IRainman** [comment]
FMT is better option for this.

**Comment by IRainman** [comment]
[fmt](https://github.com/fmtlib/fmt) I use both projects, fmt and fast_float, and they both work very well in my math physical engine. Strongly recommend  🖖🏻

**Comment by IRainman** [comment]
Now I recommend use Zmij https://github.com/vitaut/zmij if you target only print floats, it's much faster.
